ERROR - 2023-06-21 03:15:53 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'mligo'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\mligo\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\mligo\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\mligo\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\mligo\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
#7 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-06-21 03:15:53 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
ERROR - 2023-06-21 03:16:02 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'mligo'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\mligo\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\mligo\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\mligo\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(25): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->create()
#9 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\mligo\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\mligo\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\mligo\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(25): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->create()
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2023-06-21 03:16:02 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `password`) VALUES (:username:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\AuthController.php(25): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AuthController->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
ERROR - 2023-06-21 03:16:11 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'mligo'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-06-21 03:16:35 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'mligo'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-06-21 03:17:00 --> Error connecting to the database: ErrorException: mysqli::real_connect(): Error while reading greeting packet. PID=17880 in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 173)
#1 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#5 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#7 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#8 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#12 {main}

Next mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-06-21 03:19:05 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '' at line 1 in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(295): mysqli->query('SHOW TABLES FRO...', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SHOW TABLES FRO...')
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}
ERROR - 2023-06-21 03:23:08 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'mligo'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-06-21 03:23:32 --> Error connecting to the database: ErrorException: mysqli::real_connect(): Error while reading greeting packet. PID=13896 in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 173)
#1 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#5 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#7 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#8 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#12 {main}

Next mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-06-21 03:24:49 --> Error connecting to the database: ErrorException: mysqli::real_connect(): Error while reading greeting packet. PID=12672 in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 173)
#1 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#5 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#7 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#8 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#12 {main}

Next mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-06-21 03:25:22 --> Error connecting to the database: ErrorException: mysqli::real_connect(): Error while reading greeting packet. PID=9548 in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 173)
#1 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#5 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#7 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#8 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#12 {main}

Next mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
CRITICAL - 2023-06-21 03:25:38 --> You have not selected a database type to connect to.
in SYSTEMPATH\Database\Database.php on line 53.
 1 SYSTEMPATH\Database\Config.php(82): CodeIgniter\Database\Database->load([...], 'default')
 2 SYSTEMPATH\Common.php(362): CodeIgniter\Database\Config::connect('default', true)
 3 SYSTEMPATH\Database\MigrationRunner.php(151): db_connect(null)
 4 SYSTEMPATH\Config\Services.php(411): CodeIgniter\Database\MigrationRunner->__construct(null, null)
 5 SYSTEMPATH\Config\BaseService.php(252): CodeIgniter\Config\Services::migrations(Object(Config\Migrations), null, false)
 6 SYSTEMPATH\Config\BaseService.php(193): CodeIgniter\Config\BaseService::__callStatic('migrations', [...])
 7 SYSTEMPATH\Config\Services.php(406): CodeIgniter\Config\BaseService::getSharedInstance('migrations', null, null)
 8 SYSTEMPATH\Config\BaseService.php(252): CodeIgniter\Config\Services::migrations()
 9 SYSTEMPATH\Commands\Database\Migrate.php(69): CodeIgniter\Config\BaseService::__callStatic('migrations', [])
10 SYSTEMPATH\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run([])
11 SYSTEMPATH\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', [])
12 ROOTPATH\spark(97): CodeIgniter\CLI\Console->run()
ERROR - 2023-06-21 03:26:46 --> Error connecting to the database: ErrorException: mysqli::real_connect(): Error while reading greeting packet. PID=1896 in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 173)
#1 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#5 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#7 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#8 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#12 {main}

Next mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
CRITICAL - 2023-06-21 03:29:35 --> Class "CodeIgniter\Database\SQLite3 \Connection" not found
in SYSTEMPATH\Database\Database.php on line 139.
 1 SYSTEMPATH\Database\Database.php(56): CodeIgniter\Database\Database->initDriver('SQLite3 ', 'CodeIgniter\\Database\\SQLite3 \\Connection', [...])
 2 SYSTEMPATH\Database\Config.php(82): CodeIgniter\Database\Database->load([...], 'default')
 3 SYSTEMPATH\Common.php(362): CodeIgniter\Database\Config::connect('default', true)
 4 SYSTEMPATH\Database\MigrationRunner.php(151): db_connect(null)
 5 SYSTEMPATH\Config\Services.php(411): CodeIgniter\Database\MigrationRunner->__construct(null, null)
 6 SYSTEMPATH\Config\BaseService.php(252): CodeIgniter\Config\Services::migrations(Object(Config\Migrations), null, false)
 7 SYSTEMPATH\Config\BaseService.php(193): CodeIgniter\Config\BaseService::__callStatic('migrations', [...])
 8 SYSTEMPATH\Config\Services.php(406): CodeIgniter\Config\BaseService::getSharedInstance('migrations', null, null)
 9 SYSTEMPATH\Config\BaseService.php(252): CodeIgniter\Config\Services::migrations()
10 SYSTEMPATH\Commands\Database\Migrate.php(69): CodeIgniter\Config\BaseService::__callStatic('migrations', [])
11 SYSTEMPATH\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run([])
12 SYSTEMPATH\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', [])
13 ROOTPATH\spark(97): CodeIgniter\CLI\Console->run()
ERROR - 2023-06-21 03:29:49 --> Error connecting to the database: ErrorException: mysqli::real_connect(): Error while reading greeting packet. PID=11340 in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 173)
#1 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#5 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#7 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#8 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#12 {main}

Next mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-06-21 03:34:54 --> Error connecting to the database: ErrorException: mysqli::real_connect(): Error while reading greeting packet. PID=16960 in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 173)
#1 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo@localhost', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#5 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#7 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#8 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#12 {main}

Next mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo@localhost', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
CRITICAL - 2023-06-21 03:35:21 --> Class "CodeIgniter\Database\MySQL\Connection" not found
in SYSTEMPATH\Database\Database.php on line 139.
 1 SYSTEMPATH\Database\Database.php(56): CodeIgniter\Database\Database->initDriver('MySQL', 'CodeIgniter\\Database\\MySQL\\Connection', [...])
 2 SYSTEMPATH\Database\Config.php(82): CodeIgniter\Database\Database->load([...], 'default')
 3 SYSTEMPATH\Common.php(362): CodeIgniter\Database\Config::connect('default', true)
 4 SYSTEMPATH\Database\MigrationRunner.php(151): db_connect(null)
 5 SYSTEMPATH\Config\Services.php(411): CodeIgniter\Database\MigrationRunner->__construct(null, null)
 6 SYSTEMPATH\Config\BaseService.php(252): CodeIgniter\Config\Services::migrations(Object(Config\Migrations), null, false)
 7 SYSTEMPATH\Config\BaseService.php(193): CodeIgniter\Config\BaseService::__callStatic('migrations', [...])
 8 SYSTEMPATH\Config\Services.php(406): CodeIgniter\Config\BaseService::getSharedInstance('migrations', null, null)
 9 SYSTEMPATH\Config\BaseService.php(252): CodeIgniter\Config\Services::migrations()
10 SYSTEMPATH\Commands\Database\Migrate.php(69): CodeIgniter\Config\BaseService::__callStatic('migrations', [])
11 SYSTEMPATH\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run([])
12 SYSTEMPATH\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', [])
13 ROOTPATH\spark(97): CodeIgniter\CLI\Console->run()
ERROR - 2023-06-21 03:35:51 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for 178.128.123.205@localhost failed: A non-recoverable error occurred during a database lookup.  in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 173)
#1 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205...', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#5 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#7 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#8 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for 178.128.123.205@localhost failed: A non-recoverable error occurred during a database lookup.  in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205...', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for 178.128.****.205@localhost failed: A non-recoverable error occurred during a database lookup.  in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-06-21 03:36:06 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for 178.128.123.205/localhost failed: No such host is known.  in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 173)
#1 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205...', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#5 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#7 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#8 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for 178.128.123.205/localhost failed: No such host is known.  in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205...', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for 178.128.****.205/localhost failed: No such host is known.  in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-06-21 03:37:00 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'mligo'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-06-21 03:37:35 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'mligo'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\mligo\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\mligo\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\mligo\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(25): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->create()
#9 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\mligo\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\mligo\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\mligo\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(25): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->create()
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2023-06-21 03:37:35 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `password`) VALUES (:username:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\AuthController.php(25): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AuthController->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
ERROR - 2023-06-21 03:37:40 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'mligo'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\mligo\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\mligo\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\mligo\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(25): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->create()
#9 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\mligo\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\mligo\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\mligo\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(25): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->create()
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2023-06-21 03:37:40 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `password`) VALUES (:username:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\AuthController.php(25): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AuthController->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
ERROR - 2023-06-21 04:22:12 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '' at line 1 in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(295): mysqli->query('SHOW TABLES FRO...', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SHOW TABLES FRO...')
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}
CRITICAL - 2023-06-21 04:28:26 --> Invalid file: "dashboard/login.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('dashboard/login.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('dashboard/login', [], true)
 3 APPPATH\Controllers\Home.php(9): view('dashboard/login')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
CRITICAL - 2023-06-21 04:28:43 --> Invalid file: "login.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('login.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('login', [], true)
 3 APPPATH\Controllers\Home.php(9): view('login')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
ERROR - 2023-06-21 04:40:20 --> mysqli_sql_exception: No database selected in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\mligo\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\mligo\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2023-06-21 04:40:20 --> No database selected
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *
FROM `users`
WHERE `username` = \'adi\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `username` = \'adi\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `username` = \'adi\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 7 APPPATH\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
ERROR - 2023-06-21 04:40:37 --> mysqli_sql_exception: No database selected in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\mligo\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\mligo\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2023-06-21 04:40:37 --> No database selected
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *
FROM `users`
WHERE `username` = \'adi\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `users`
WHERE `username` = \'adi\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `users`
WHERE `username` = \'adi\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 7 APPPATH\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
ERROR - 2023-06-21 04:40:51 --> mysqli_sql_exception: No database selected in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\mligo\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\mligo\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\mligo\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(25): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->create()
#9 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-06-21 04:40:51 --> No database selected
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `users` (`username`, `password`) VALUES (\'1234\', \'$2y$10$99yrLrbpKC.cXqJ826jIPee6T08YAl.PzRflrfScfXyJ1r8MkdTRC\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `password`) VALUES (\'1234\', \'$2y$10$99yrLrbpKC.cXqJ826jIPee6T08YAl.PzRflrfScfXyJ1r8MkdTRC\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `password`) VALUES (\'1234\', \'$2y$10$99yrLrbpKC.cXqJ826jIPee6T08YAl.PzRflrfScfXyJ1r8MkdTRC\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `password`) VALUES (:username:, :password:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\AuthController.php(25): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AuthController->create()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
ERROR - 2023-06-21 11:57:25 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'login', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\mligo\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\mligo\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\mligo\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\mligo\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
#7 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-06-21 11:57:25 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
ERROR - 2023-06-21 12:03:38 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'login', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\mligo\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\mligo\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\mligo\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(25): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->create()
#9 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\mligo\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\mligo\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\mligo\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(25): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->create()
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2023-06-21 12:03:38 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `password`) VALUES (:username:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\AuthController.php(25): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AuthController->create()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
ERROR - 2023-06-21 12:04:18 --> Error connecting to the database: ErrorException: mysqli::real_connect(): Error while reading greeting packet. PID=19052 in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 173)
#1 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#5 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#7 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#8 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#12 {main}

Next mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-06-21 12:05:06 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), 'login', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-06-21 12:05:26 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'login', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
CRITICAL - 2023-06-21 12:28:54 --> Class "App\Models\LoginModel" not found
in APPPATH\Controllers\AuthController.php on line 15.
 1 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\AuthController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
CRITICAL - 2023-06-21 12:28:56 --> Class "App\Models\LoginModel" not found
in APPPATH\Controllers\AuthController.php on line 15.
 1 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\AuthController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
CRITICAL - 2023-06-21 12:29:19 --> Class "App\Models\LoginModel" not found
in APPPATH\Controllers\AuthController.php on line 15.
 1 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\AuthController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
CRITICAL - 2023-06-21 12:29:20 --> Class "App\Models\LoginModel" not found
in APPPATH\Controllers\AuthController.php on line 15.
 1 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\AuthController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
CRITICAL - 2023-06-21 12:32:11 --> Class "App\Models\LoginModel" not found
in APPPATH\Controllers\AuthController.php on line 15.
 1 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\AuthController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
CRITICAL - 2023-06-21 12:32:14 --> Class "App\Models\LoginModel" not found
in APPPATH\Controllers\AuthController.php on line 15.
 1 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\AuthController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
CRITICAL - 2023-06-21 12:32:26 --> Class "App\Models\LoginModel" not found
in APPPATH\Controllers\AuthController.php on line 15.
 1 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\AuthController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
CRITICAL - 2023-06-21 12:33:51 --> Class "App\Models\LoginModel" not found
in APPPATH\Controllers\AuthController.php on line 15.
 1 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\AuthController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
CRITICAL - 2023-06-21 12:40:53 --> Class "App\Models\LoginModel" not found
in APPPATH\Controllers\AuthController.php on line 15.
 1 SYSTEMPATH\CodeIgniter.php(906): App\Controllers\AuthController->__construct()
 2 SYSTEMPATH\CodeIgniter.php(490): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
CRITICAL - 2023-06-21 12:53:05 --> Cannot access offset of type string on string
in APPPATH\Controllers\AuthController.php on line 51.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
CRITICAL - 2023-06-21 12:53:47 --> Cannot access offset of type string on string
in APPPATH\Controllers\AuthController.php on line 51.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
CRITICAL - 2023-06-21 12:53:50 --> Cannot access offset of type string on string
in APPPATH\Controllers\AuthController.php on line 51.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
CRITICAL - 2023-06-21 12:54:22 --> Cannot access offset of type string on string
in APPPATH\Controllers\AuthController.php on line 51.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
CRITICAL - 2023-06-21 12:54:25 --> Cannot access offset of type string on string
in APPPATH\Controllers\AuthController.php on line 51.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
CRITICAL - 2023-06-21 12:54:40 --> Cannot access offset of type string on string
in APPPATH\Controllers\AuthController.php on line 51.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
ERROR - 2023-06-21 13:06:51 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'mligo'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'mligo', Object(SensitiveParameterValue), 'mlijo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-06-21 13:07:49 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'mligo'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-06-21 13:08:03 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'Mligo'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'Mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-06-21 13:11:33 --> Error connecting to the database: ErrorException: mysqli::real_connect(): Error while reading greeting packet. PID=15596 in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 173)
#1 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#5 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#7 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#8 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#12 {main}

Next mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-06-21 13:14:07 --> Error connecting to the database: ErrorException: mysqli::real_connect(): Error while reading greeting packet. PID=7780 in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 173)
#1 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo@localhost', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#5 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#7 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#8 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#12 {main}

Next mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo@localhost', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-06-21 13:21:55 --> Error connecting to the database: ErrorException: mysqli::real_connect(): Error while reading greeting packet. PID=16520 in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 173)
#1 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo@localhost', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp\htdocs\mligo\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\mligo\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#7 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(48): CodeIgniter\BaseModel->first()
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
#9 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo@localhost', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\mligo\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\mligo\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(48): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\mligo\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\mligo\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(48): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
#7 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-06-21 13:21:55 --> Unable to connect to the database.
Main connection [MySQLi]: MySQL server has gone away
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `Login`
WHERE `Username` = :Username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthController.php(48): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
ERROR - 2023-06-21 13:22:01 --> Error connecting to the database: ErrorException: mysqli::real_connect(): Error while reading greeting packet. PID=16520 in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 173)
#1 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo@localhost', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp\htdocs\mligo\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\mligo\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#7 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(48): CodeIgniter\BaseModel->first()
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
#9 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('178.128.123.205', 'mligo@localhost', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\mligo\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\mligo\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(48): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\mligo\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\mligo\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(48): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
#7 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-06-21 13:22:01 --> Unable to connect to the database.
Main connection [MySQLi]: MySQL server has gone away
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `Login`
WHERE `Username` = :Username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthController.php(48): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
ERROR - 2023-06-21 13:22:41 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'mligo' in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\mligo\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\mligo\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(48): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'mligo' in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\mligo\system\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\mligo\system\BaseModel.php(656): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\mligo\app\Controllers\AuthController.php(48): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\mligo\system\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
#7 C:\xampp\htdocs\mligo\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#8 C:\xampp\htdocs\mligo\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\mligo\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\mligo\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-06-21 13:22:41 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'mligo'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `Login`
WHERE `Username` = :Username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(268): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(656): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthController.php(48): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\AuthController->authenticate()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
ERROR - 2023-06-21 13:23:03 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'mligo' in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'mligo', 3306, '', 0)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'mligo' in C:\xampp\htdocs\mligo\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1409): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\mligo\system\Database\BaseConnection.php(1428): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\mligo\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\mligo\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\mligo\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\mligo\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\mligo\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
CRITICAL - 2023-06-21 13:39:42 --> Undefined variable $user
in APPPATH\Views\dashboard\profile.php on line 21.
 1 APPPATH\Views\dashboard\profile.php(21): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $user', 'C:\\xampp\\htdocs\\mligo\\app\\Views\\dashboard\\profile.php', 21)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\mligo\\app\\Views\\dashboard\\profile.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('dashboard/profile', [], true)
 5 APPPATH\Controllers\Dashboard.php(27): view('dashboard/profile')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Dashboard->profile()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
CRITICAL - 2023-06-21 13:39:58 --> Undefined variable $user
in APPPATH\Views\dashboard\profile.php on line 21.
 1 APPPATH\Views\dashboard\profile.php(21): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $user', 'C:\\xampp\\htdocs\\mligo\\app\\Views\\dashboard\\profile.php', 21)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\mligo\\app\\Views\\dashboard\\profile.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('dashboard/profile', [], true)
 5 APPPATH\Controllers\Dashboard.php(27): view('dashboard/profile')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Dashboard->profile()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
CRITICAL - 2023-06-21 13:40:02 --> Undefined variable $user
in APPPATH\Views\dashboard\profile.php on line 21.
 1 APPPATH\Views\dashboard\profile.php(21): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $user', 'C:\\xampp\\htdocs\\mligo\\app\\Views\\dashboard\\profile.php', 21)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\mligo\\app\\Views\\dashboard\\profile.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('dashboard/profile', [], true)
 5 APPPATH\Controllers\Dashboard.php(27): view('dashboard/profile')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Dashboard->profile()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
CRITICAL - 2023-06-21 13:40:09 --> Undefined variable $user
in APPPATH\Views\dashboard\profile.php on line 21.
 1 APPPATH\Views\dashboard\profile.php(21): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $user', 'C:\\xampp\\htdocs\\mligo\\app\\Views\\dashboard\\profile.php', 21)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\mligo\\app\\Views\\dashboard\\profile.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('dashboard/profile', [], true)
 5 APPPATH\Controllers\Dashboard.php(27): view('dashboard/profile')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Dashboard->profile()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\mligo\\public\\index.php')
